src/main/java/net/openhft/chronicle/network/TcpEventHandler.java

```diff
@@ -59,6 +59,7 @@
         implements EventHandler, TcpEventHandlerManager<T> {
 
     public static final int TARGET_WRITE_SIZE = Integer.getInteger("TcpEventHandler.targetWriteSize", 1024);
+    private static final boolean CALL_MISSED_HEARTBEAT_ON_DISCONNECT = Jvm.getBoolean("chronicle.network.callOnMissedHeartbeatOnDisconnect");
```

Comment:
`callOnMissedHeartbeatOnDisconnect` should this be `callMissedHeartbeatOnDisconnect`

Reply:
Actually no I see now